src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -132,7 +208,7 @@ private function getContainerDeprecationLogs(): array
         $logs = [];
         foreach (unserialize($logContent) as $log) {
             $log['context'] = ['exception' => new SilencedErrorContext($log['type'], $log['file'], $log['line'], $log['trace'], $log['count'])];
-            $log['timestamp'] = $bootTime;
+            $log['timestamp'] = (new \DateTimeImmutable())->setTimestamp($bootTime)->format(\DateTimeInterface::RFC3339_EXTENDED);
```

Comment:
shouldn't this set the `timestamp_rfc3339` key instead ?

Reply:
I remember having some problems about this ... and this was the only solution that worked always (the  `timestamp_rfc3339` key was not always present or something like that). I don't remember the details, sorry!